packages/react-instantsearch-core/src/core/utils.js

```diff
@@ -44,3 +44,18 @@ export const removeEmptyKey = obj => {
 
   return obj;
 };
+
+export function addAbsolutePositions(hits, hitsPerPage, page) {
+  return hits.map((hit, index) => ({
+    ...hit,
+    __position: hitsPerPage * page + index + 1,
+  }));
+}
+
+export function addQueryID(hits, queryID) {
+  if (!queryID) return hits;
```

Comment:
Can we avoid inline returns?

Reply:
I know this is a pretty common coding style preference but it doesn't seem we follow it in the codebase: 
eg. https://github.com/algolia/react-instantsearch/blob/ec9e0fbd6106d1c3e47f1dbfa6eaac3e20af6bd5/packages/react-instantsearch-core/src/components/Index.js#L82

If we decide this is a coding style we want to adopt, we should just add it to linting. 
https://eslint.org/docs/rules/curly#require-following-curly-brace-conventions-curly

Reply:
It's mentioned in our [ESLint config](https://github.com/algolia/eslint-config-algolia/blob/e6661db323a69d4304a4bc84571e2263e572a6a4/rules/base.js#L45) but for some reason not picked up.

We can still apply this rule for new code.

Reply:
of course, but it's best to leverage the tooling to move the focus away from style in reviews.

I looked a bit and it looks like it's disabled by prettier which we use in algolia/react
https://github.com/prettier/eslint-config-prettier/blob/master/index.js#L9

We should reenable it if we care about this curly rule

Reply:
this is fixed btw, but looking forward to continue the discussion on adding this rule to our linter.